docs/ruleset_functions/get_zone_BPF_BAT.md

```diff
@@ -0,0 +1,142 @@
+	LOBBY_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	LOBBY_ELEVATOR: UNDETERMINED,
+	LOBBY_HOTEL: HOTEL_MOTEL,
+	LOBBY_MOTION_PICTURE_THEATER: ALL_OTHER,
+	LOBBY_PERFORMING_ARTS_THEATER: ALL_OTHER,
+	LOBBY_ALL_OTHERS: UNDETERMINED,
+	LOCKER_ROOM: UNDETERMINED,
+	LOUNGE_BREAKROOM_HEALTH_CARE_FACILITY: HEALTHCARE_HOSPITAL,
+	LOUNGE_BREAKROOM_ALL_OTHERS: UNDETERMINED,
+	OFFICE_ENCLOSED: OFFICE,
+	OFFICE_OPEN_PLAN: OFFICE,
+	PARKING_AREA_INTERIOR: UNDETERMINED,
+	PHARMACY_AREA: UNDETERMINED,
+	RESTROOM_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	RESTROOM_ALL_OTHERS: UNDETERMINED,
+	SALES_AREA: RETAIL,
+	SEATING_AREA_GENERAL: UNDETERMINED,
+	STAIRWELL: UNDETERMINED,
+	STORAGE_ROOM_HOSPITAL: HEALTHCARE_HOSPITAL,
+	STORAGE_ROOM_SMALL: UNDETERMINED,
+	STORAGE_ROOM_LARGE: UNDETERMINED,
+	VEHICULAR_MAINTENANCE_AREA: ALL_OTHER,
+	WORKSHOP: UNDETERMINED,
+	ASSISTED_LIVING_FACILITY_CHAPEL: UNDETERMINED,
+	ASSISTED_LIVING_FACILITY_RECREATION_ROOM_COMMON_LIVING_ROOM: UNDETERMINED,
+	CONVENTION_CENTER_EXHIBIT_SPACE: ALL_OTHER,
+	DORMITORY_LIVING_QUARTERS: MULTIFAMILY,  
+	FIRE_STATION_SLEEPING_QUARTERS: UNDETERMINED,
+	GYMNASIUM_FITNESS_CENTER_EXERCISE_AREA: UNDETERMINED,
+	GYMNASIUM_FITNESS_CENTER_PLAYING_AREA: UNDETERMINED,
+	HEALTHCARE_FACILITY_EMERGENCY_ROOM: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_EXAM_TREATMENT_ROOM: HEALTHCARE_HOSPITAL,
+    HEALTHCARE_FACILITY_MEDICAL_SUPPLY_ROOM: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_NURSERY: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_NURSES_STATION: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_OPERATING_ROOM: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_PATIENT_ROOM: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_PHYSICAL_THERAPY_ROOM: HEALTHCARE_HOSPITAL,
+	HEALTHCARE_FACILITY_RECOVERY_ROOM: HEALTHCARE_HOSPITAL,
+	LIBRARY_READING_AREA: UNDETERMINED,
+	LIBRARY_STACKS: UNDETERMINED,
+	MANUFACTURING_FACILITY_DETAILED_MANUFACTURING_AREA: ALL_OTHER,
+	MANUFACTURING_FACILITY_EQUIPMENTROOM: ALL_OTHER,
+	MANUFACTURING_FACILITY_EXTRA_HIGH_BAY_AREA: ALL_OTHER,
+	MANUFACTURING_FACILITY_HIGH_BAY_AREA: ALL_OTHER,
+	MANUFACTURING_FACILITY_LOW_BAY_AREA: ALL_OTHER,
+	MUSEUM_GENERAL_EXHIBITION_AREA: ALL_OTHER,
+	MUSEUM_RESTORATION_ROOM: ALL_OTHER,
+	POST_OFFICE_SORTING_AREA: ALL_OTHER,
+	RELIGIOUS_FACILITY_FELLOWSHIP_HALL: ALL_OTHER,
+	RELIGIOUS_FACILITY_WORSHIP_PULPIT_CHOIR_AREA: ALL_OTHER,
+	RETAIL_FACILITIES_DRESSING_FITTING_ROOM: RETAIL,
+	RETAIL_FACILITIES_MALL_CONCOURSE: RETAIL,
+	SPORTS_ARENA_PLAYING_AREA_CLASS_I_FACILITY: ALL_OTHER,
+	SPORTS_ARENA_PLAYING_AREA_CLASS_II_FACILITY: ALL_OTHER,
+	SPORTS_ARENA_PLAYING_AREA_CLASS_III_FACILITY: ALL_OTHER,
+	SPORTS_ARENA_PLAYING_AREA_CLASS_IV_FACILITY: UNDETERMINED,
+	TRANSPORTATION_FACILITY_BAGGAGE_CAROUSEL_AREA: ALL_OTHER,
+	TRANSPORTATION_FACILITY_AIRPORT_CONCOURSE: ALL_OTHER,
+	TRANSPORTATION_FACILITY_TICKET_COUNTER: ALL_OTHER,
+	WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS: WAREHOUSE,
+	WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS: WAREHOUSE,
```

Comment:
is the correct classification `WAREHOUSE `or `HEATED-ONLY STORAGE`? There are different classifications between `get_zone_BPF_BAT `and `get_BPF_building_area_types_and_zones`

Reply:
correct classification is `WAREHOUSE`. Removed `HEATED-ONLY STORAGE`